zerver/lib/emoji.py

```diff
@@ -117,6 +117,6 @@ def get_emoji_url(emoji_file_name: Text, realm_id: int) -> Text:
     return upload_backend.get_emoji_url(emoji_file_name, realm_id)
 
 
-def get_emoji_file_name(emoji_file_name: Text, emoji_name: Text) -> Text:
+def get_emoji_file_name(emoji_file_name: Text, emoji_id: int) -> Text:
     _, image_ext = os.path.splitext(emoji_file_name)
-    return ''.join((emoji_name, image_ext))
+    return ''.join((str(emoji_id), image_ext))
```

Comment:
for the filenames maybe we should do `<emoji_name>-<emoji_id>.<file_ext>`?

Reply:
I think we just want the ID.  That way, if we later want to add an "edit realm emoji" feature, where an admin can change the emoji's name afterwards, we don't need to do another migration.

Reply:
> for the filenames maybe we should do <emoji_name>-<emoji_id>.<file_ext>?

Any particular benefit of using this filename format?

Reply:
Not really as Tim pointed out.